assets/brunch-config.js

```diff
@@ -0,0 +1,70 @@
+const neat = require('bourbon-neat').includePaths;
+const bourbon = require('bourbon').includePaths;
+
+exports.config = {
+  // See http://brunch.io/#documentation for docs.
+  files: {
+    javascripts: {
+      joinTo: "js/app.js"
+
+      // To use a separate vendor.js bundle, specify two files path
+      // http://brunch.io/docs/config#-files-
+      // joinTo: {
+      //   "js/app.js": /^js/,
+      //   "js/vendor.js": /^(?!js)/
+      // }
+      //
+      // To change the order of concatenation of files, explicitly mention here
+      // order: {
+      //   before: [
+      //     "vendor/js/jquery-2.1.1.js",
+      //     "vendor/js/bootstrap.min.js"
+      //   ]
+      // }
+    },
+    stylesheets: {
+      joinTo: "css/app.css"
+    },
+    templates: {
+      joinTo: "js/app.js"
```

Comment:
Strings must use singlequote.

[Read more about it here.](http://eslint.org/docs/rules/quotes)